pkg/registry/daemoncontroller/doc.go

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
```

Comment:
It should only be changed in new files, existing files should keep the date
they were started with.

On Wed, Jul 8, 2015 at 4:21 PM, Ananya Kumar <notifications@github.com>
wrote:

> In pkg/registry/daemoncontroller/doc.go
> <https://github.com/vmarmol/kubernetes/pull/1#discussion_r34209841>:
>
> > @@ -0,0 +1,19 @@
> > +/*
> > +Copyright 2014 The Kubernetes Authors All rights reserved.
>
> Whoops :P I'll change this in all the files I added/edited.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vmarmol/kubernetes/pull/1/files#r34209841>.
>